python/gtsam/specializations.h

```diff
@@ -13,3 +13,6 @@ py::bind_vector<std::vector<gtsam::BinaryMeasurement<gtsam::Unit3> > >(m_, "Bina
 py::bind_map<gtsam::IndexPairSetMap>(m_, "IndexPairSetMap");
 py::bind_vector<gtsam::IndexPairVector>(m_, "IndexPairVector");
 py::bind_map<gtsam::KeyPairDoubleMap>(m_, "KeyPairDoubleMap");
+py::bind_vector<std::vector<gtsam::SfmMeasurement> >(m_, "Measurement");
+py::bind_vector<std::vector<gtsam::SiftIndex> >(m_, "SiftIndexVector");
+py::bind_vector<std::vector<gtsam::SfmCamera> >(m_, "cameras");
```

Comment:
Do we need vector cameras? Maybe we could call them VectorPinholeCameraCal3Bundler after that PR has been merged. Or maybe in that PR itself, if we need them.